Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1678,6 +1681,9 @@ RcTcacheEntry TextureCacheBase::GetTexture(const int textureCacheSafetyColorSamp
   return entry;
 }
 
+// Note: the following function assumes all CustomTextureData
+// has a single slice.  This is verified with the 'GameTexture::Validate' function
+// after the data is loaded
```

Comment:
Is there a reason cube maps can't work with this, or that we don't want them to? If so that would probably be worth adding to this comment.

Reply:
Cube maps require a cube-map specific texture sampler to be present, this concept doesn't exist for the GC/Wii (or at least wasn't implemented in the same way) so these don't make sense here.  Textures are loaded into a texture array.  While a cubemap is technically just an array of textures (with special meaning) it doesn't really make sense in this context.  I added a comment but didn't elaborate on cubemaps specifically, let me know if you think it's relevant.

Reply:
The comment seems fine to me now. The formatting's a bit cut up though; it can be condensed from 5 lines to 3.

Reply:
Should be good now!